app/templates/components/forms/orders/order-form.hbs

```diff
@@ -65,7 +65,7 @@
             </div>
           {{/if}}
         {{/each}}
-        
+
```

Comment:
Remove this empty line. Rest looks good.

Reply:
@pradeepgangwar Made the changes please review